.github/workflows/pytest.yaml

```diff
@@ -2,8 +2,6 @@ name: Python package tests
 
 on:
   push:
-  pull_request:
-    types: [opened, synchronize, reopened]
```

Comment:
The presence of this event was causing the workflow to be run twice.

Reply:
yea, althought I would like the PR to run atleast once, I think taking it out, might stop it all together. 

Do you mind putting it back in, and then we could do a seperate PR for this?

Reply:
Yep, put that back in.